# HG changeset patch
# User Rik <rik@octave.org>
# Date 1360422879 28800
#      Sat Feb 09 07:14:39 2013 -0800
# Node ID a78c1e445cfc2b290d8ca8c84ea1e179f48400a8
# Parent  98f90d4dca20df3035deeaffcd8a55bc8a6a7f59
Change default history size to 1000.
Command number in prompt will now easily match command number in history.

* libinterp/interpfcn/oct-hist.cc(default_history_size): Change size from
1024 to 1000.

* libinterp/interpfcn/oct-hist.cc(Fhistory_size): Change docstring to
say default history size is 1000.

diff --git a/libinterp/interpfcn/oct-hist.cc b/libinterp/interpfcn/oct-hist.cc
--- a/libinterp/interpfcn/oct-hist.cc
+++ b/libinterp/interpfcn/oct-hist.cc
@@ -86,17 +86,17 @@ default_history_file (void)
                              ".octave_hist");
 
   return file;
 }
 
 static int
 default_history_size (void)
 {
-  int size = 1024;
+  int size = 1000;
 
   std::string env_size = octave_env::getenv ("OCTAVE_HISTSIZE");
 
   if (! env_size.empty ())
     {
       int val;
 
       if (sscanf (env_size.c_str (), "%d", &val) == 1)
@@ -730,17 +730,17 @@ the history list, subject to the value o
   return retval;
 }
 
 DEFUN (history_size, args, nargout,
   "-*- texinfo -*-\n\
 @deftypefn  {Built-in Function} {@var{val} =} history_size ()\n\
 @deftypefnx {Built-in Function} {@var{old_val} =} history_size (@var{new_val})\n\
 Query or set the internal variable that specifies how many entries\n\
-to store in the history file.  The default value is @code{1024},\n\
+to store in the history file.  The default value is @code{1000},\n\
 but may be overridden by the environment variable @w{@env{OCTAVE_HISTSIZE}}.\n\
 @seealso{history_file, history_timestamp_format_string, saving_history}\n\
 @end deftypefn")
 {
   int old_history_size = command_history::size ();
 
   int tmp = old_history_size;
 
